File: py/Soal-2/mamalia.py
from makhluk import Makhluk

class Mamalia(Makhluk):
    def __init__(self) :
        super().__init__("Mamalia")
        self.__nama=""
        self.__ukuran=""
        self.__jumlahKaki=0
        self.__habitat=""

    def setNama(self, nama) :
        self.__nama=nama

    def setUkuran(self, ukrn) :
        self.__ukuran=ukrn

    def setJumlahKaki(self, jmlKaki) :
        self.__jumlahKaki=jmlKaki

    def setHabitat(self, habitat) :
        self.__habitat=habitat

    def getNama(self) :
        return self.__nama

    def getUkuran(self) :
        return self.__ukuran

    def getJumlahKaki(self) :
        return self.__jumlahKaki

    def getHabitat(self) :
        return self.__habitat

    def printInfo(self) :
        super().printInfo()
        print("Nama\t\t:" + self.__nama)
        print("Ukuran\t\t:" + self.__ukuran)
        print("Jumlah Kaki\t:" + str(self.__jumlahKaki))
        print("Habitat\t\t:" + self.__habitat)

File: java/Soal-2/Mamalia.java
public class Mamalia extends Makhluk {
    private String nama;
    private String ukuran;
    private int jumlahKaki;
    private String habitat;

    Mamalia() {
        super.setJenis("Mamalia");
        this.nama="";
        this.ukuran="";
        this.jumlahKaki=0;
        this.habitat="";
    }

    Mamalia(String nama, String ukuran, int jumlahKaki, String habitat) {
        super.setJenis("Mamalia");
        this.nama=nama;
        this.ukuran=ukuran;
        this.jumlahKaki=jumlahKaki;
        this.habitat=habitat;
    }

    public void setNama(String nama) {
        this.nama=nama;
    }

    public void setUkuran(String ukrn) {
        this.ukuran=ukrn;
    }

    public void setJumlahKaki(int jmlKaki) {
        this.jumlahKaki=jmlKaki;
    }

    public void setHabitat(String habitat) {
        this.habitat=habitat;
    }

    public String getNama() {
        return nama;
    }

    public String getUkuran() {
        return ukuran;
    }

    public int getJumlahKaki() {
        return jumlahKaki;
    }

    public String getHabitat() {
        return habitat;
    }

    public void printInfo() {
        super.printInfo();
        System.out.println("Nama\t\t:" + this.nama);
        System.out.println("Ukuran\t\t:" + this.ukuran);
        System.out.println("Jumlah Kaki\t:" + this.jumlahKaki);
        System.out.println("Habitat\t\t:" + this.habitat);
    }
}
